plugins/modules/junos_prefix_lists.py

```diff
@@ -606,11 +639,10 @@
           - 172.16.7.32
           - 172.16.9.32
     state: rendered
-#
-#
-# -------------------------
-# Module Execution Result
-# -------------------------
+
+
+# Task Output
+# -------------
```

Comment:
please add left out operations in Return Section
ref: https://github.com/ansible-collections/junipernetworks.junos/blob/6f75fdad3f0a2d7dc3378b22f58943f26f406fc2/plugins/modules/junos_bgp_address_family.py#L1954